### DISCUSSION TITLE
请问biominer-aget是否有断点续传功能

### DISCUSSION THREAD

**Discussion by Ambious3** [chinese-quartet/docs.chinese-quartet.org#3] [Q&A] [answered]
实际使用biominer-aget下载数据有时会卡死，重新运行命令会提示fastq exists, please remove it and retry!，只能选择删除下载一半的数据重新开始下载吗

**Comment by yjcyxky** [comment, marked as answer]
> 实际使用biominer-aget下载数据有时会卡死，重新运行命令会提示fastq exists, please remove it and retry!，只能选择删除下载一半的数据重新开始下载吗

您好，抱歉回复晚了以及给您带来的困扰。biominer-aget是有断点续传功能的，但应该是因为当前默认的仓库Node不支持分片下载，导致无法启用断点续传功能。我们目前已经将数据托管至支持分片下载的数据仓库GSA，近期会将其更新发布到Quartet Data Portal，届时即可启用断点续传功能。

感谢您的支持。

**Reply by Ambious3** [reply]
了解，非常感谢
